packages/lib/src/components/internal/Await/types.ts

```diff
@@ -27,4 +27,5 @@ export interface AwaitComponentProps {
     messageText: string;
     awaitText: string;
     ref: any;
+    getI18n?: (key: string, options?) => string; // Should equate to the Language.get method
```

Comment:
Is this needed? I don't see it being used

Reply:
you're right it got removed in a recent change to this PR